bamboo/set-bamboo-env-variables.sh

```diff
@@ -144,7 +144,7 @@ fi
 # Target master by default.
 # Update with appropriate conditional
 # when creating a feature branch.
-export PR_BRANCH=master
+export PR_BRANCH='feature/rds-phase-2'
```

Comment:
Should we change this back to `master`? 🤔

Reply:
Good catch, yes.   I'll set a PR to update it.

Reply:
https://github.com/nasa/cumulus/pull/2896